tests/test-ses.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const fs = require("fs");
+const httpMocks = require("node-mocks-http");
+const path = require("path");
+const test = require("tape-async");
+
+const DB = require("../db/DB");
+const getSha1 = require("../sha1-utils");
+const ses = require("../controllers/ses");
+
+
+const createRequestBody = function(notificationType) {
+  const cwd = path.join(path.dirname(fs.realpathSync(__filename)));
+  const bodyFile = path.join(cwd, `ses-${notificationType}-notification.json`);
+  return fs.readFileSync(bodyFile).toString();
+};
```

Comment:
This feels weird to me... Because you're just loading a JSON file in the current directory, me thinks you can just cheat and use something adjacent to:

```js
const createRequestBody = (notificationType) => require(`./ses-${notificationType}-notification.json`);
```

Node should also do some caching on that `require()`, which may or may not be good. Conversely if we know there are only 2 notification types, we could just store them ourselves in an Object or Map() or something:

```js
const notifications = new Map();
notifications.set("bounce", require("./ses-bounce-notification.json"));
notifications.set("complaint", require("./ses-complaint-notification.json"));
```

If you want more flexibility, we could move those JSONy blobs into a tests/fixtures/*.json folder and just load them dynamically using [**glob**](http://npm.im/glob) or some other method of reading the file system and doing whatever you want.

Reply:
Ooo, good idea! Drops the `fs` and `path` stuff altogether.